net/ostiary/files/ostiaryd.init

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=85
+
+USE_PROCD=1
+
+start_service()
+{
+
+	procd_open_instance
+	procd_set_param respawn
+	procd_set_param command /usr/bin/ostiaryd -c /etc/ostiary.cfg 
+	procd_close_instance
+}
+
+stop_service()
```

Comment:
Do we really need this? 'stop_service' will be called after procd has killed the daemon/service.

Reply:
removed